numpy/__init__.py

```diff
@@ -99,14 +90,11 @@
 Exceptions to this rule are documented.
 
 """
+import os
 import sys
 import warnings
 
 from ._globals import _NoValue, _CopyMode
-# These exceptions were moved in 1.25 and are hidden from __dir__()
-from .exceptions import (
-    ComplexWarning, ModuleDeprecationWarning, VisibleDeprecationWarning,
-    TooHardError, AxisError)
```

Comment:
I've got one question regarding an implementation detail: `VisibleDeprecationWarning` and `AxisError` are used in SciPy ~10 different files - How should it be done:
1. Have this piece of code checking NumPy version in every file?
2. Put this `if else` import in some `scipy.misc` or `utils`, and import the name from there? It would avoid code repetition (and considering that more deprecations will come, it might host such imports)